app/models/item_season.rb

```diff
@@ -9,4 +9,33 @@ class ItemSeason < ApplicationRecord
   belongs_to :produce_item
 
   validates :month_index, :country_code, presence: true
+
+  def month(month_index)
```

Comment:
@tcione if this makes any sense, I thought it could be simplified by something like this:
```
def month(month_index)
months = %w[january february march april may june july august september october november december]
months[month_index - 1]
end
```

Reply:
Generally this is better handled via i18n (internationalization). Luckily Rails does already provide month names in its default strings, so this method could become simply:
```ruby
def month(index) = I18n.t("date.month_names")[index].downcase
``` 

A nice thing is that we can easily translate the app in the future adding something like https://github.com/svenfuchs/rails-i18n

OBS: I've used the [endless method definition](https://www.bigbinary.com/blog/ruby-3-adds-endless-method-definition) introduced in Ruby 3.

Reply:
why use endless method here?